includes/payment-methods/class-upe-payment-method.php

```diff
@@ -134,9 +134,11 @@ public function get_currencies() {
 	 * Returns boolean dependent on whether payment method
 	 * can be used at checkout
 	 *
+	 * @param string $account_country Country of merchants account.
+	 *
 	 * @return bool
 	 */
-	public function is_enabled_at_checkout() {
+	public function is_enabled_at_checkout( string $account_country ) {
```

Comment:
For future cases we might need a shopper's country here as well.

Reply:
It's impossible to get a shopper's country reliably. If we have saved customer then maybe, but otherwise we don't know the country until shopper hasn't entered his billing details. That's why we hide the PM in the checkout for Affirm based on billing country using JS.

Reply:
+1, e.g. for first-time checkout (or guest checkout) we have no country available.